profiles/README.md

```diff
@@ -118,6 +118,27 @@ git lfs pull
 dvc pull
 ```
 
+**Note:** The DVC cache is stored in an AWS S3 bucket. 
```

Comment:
Can you comment a bit more on this requirement? Is this different than the dvc access troubleshooting discussed in #71?

Reply:
I've updated this now. LMK if this is not clear.

There might be better ways of handling this https://discuss.dvc.org/t/public-read-only-s3-remote/355/3

Reply:
still a bit confused. Are we providing too much detail? Why do we need to specify a minimum IAM?

Can't we just say:

```md
**Note:** The DVC cache is stored in a public AWS S3 bucket.
You must have aws cli installed and configured.
```

If folks have trouble accessing still, then they should be pointed to #71 (and possibly update instructions there)

This is my view! If you feel differently, then you can merge

Reply:
Good point about avoiding clutter where possible. I've moved it into a <details> block now.  I think the details are still useful because we encountered this issue just last week: we create a dummy AWS account (with no policy attached) so that our collaborators could access a public DVC cache. But that didn't work! This might be a relatively frequent scenario – where people create an AWS account only to access our data and create a dummy IAM to do so. I'll go ahead and merge because the clutter is gone